src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -915,6 +922,32 @@ public void UpdateAutoEvoResults(string results, string external)
         externalEffectsLabel.Text = external;
     }
 
+    public void ShowOrganelleMenu(OrganelleTemplate selectedOrganelle)
+    {
+        organelleMenu.SelectedOrganelle = selectedOrganelle;
```

Comment:
I think if an organelle is being currently moved, this method should return without opening the organelle menu (and probably also play the invalid place sound).